webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-grading.js

```diff
@@ -240,9 +240,9 @@ export class SakaiRubricGrading extends RubricsElement {
 
     const value = e.target.value;
 
-    const parsed = parseFloat(value.replace(/,/g, "."));
+    const parsed = value.replace(/,/g, ".");
 
-    if (isNaN(parsed)) {
+    if (isNaN(parseFloat(parsed))) {
```

Comment:
Why this bit? It looks identical.

Reply:
Here I'm assigning the string value to the `parsed` variable
(that will be used to set the input value, may contain the dot at the end)

And after that, I'm checking if that string can be parsed to a number